scripts/ui.js

```diff
@@ -0,0 +1,136 @@
+const containerList = document.querySelector('.tile-list');
+const basketList = document.querySelector('.basket-list');
+const placeOrderButton = document.querySelector('.btn-place-order');
+
+// Fetching data.
+async function getData() {
+    try {
+        const response = await fetch('https://raw.githubusercontent.com/alexsimkovich/patronage/main/api/data.json');
+        const data = await response.json();
```

Comment:
tutaj `await` nie jest potrzebny. Ewentualnie, można to skrócić:
```
const response = await fetch('https://raw.githubusercontent.com/alexsimkovich/patronage/main/api/data.json');
return response.json();
```

Reply:
Zmodyfikowałem ten fragment - dziękuję!